crates/sui-framework/sources/coin.move

```diff
@@ -27,6 +31,25 @@ module sui::coin {
         balance: Balance<T>
     }
 
+    /// Each Coin<T> will have a unique instance of CoinMetadata<T> that
+    /// stores the metadata for this coin type.
+    struct CoinMetadata<phantom T> has key, store {
+        id: UID,
+        /// Number of decimal places the coin uses.
+        /// A coin with `value ` N and `decimals` D should be shown as N / 10^D
+        /// E.g., a coin with `value` 7002 and decimals 3 should be displayed as 7.002
+        /// This is metadata for display usage only.
+        decimals: u8,
+        /// Name for the token
+        name: string::String,
+        /// Symbol for the token
+        symbol: ascii::String,
+        /// Description of the token
+        description: string::String,
+        /// URL for the token logo
+        url: Option<Url>
```

Comment:
Should this be called `icon_url` to make it obvious that this isn't the URL for the token itself?

Reply:
done